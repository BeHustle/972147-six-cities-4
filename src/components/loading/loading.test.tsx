import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Loading from './loading';

it(`Render loading screen`, () => {
  const tree = renderer
    .create(<Loading />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
