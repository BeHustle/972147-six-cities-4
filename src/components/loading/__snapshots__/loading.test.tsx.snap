// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render loading screen 1`] = `
<h1>
  Loading...
</h1>
`;
